tb/cocotb/soc/Makefile

```
@@ -113,10 +123,28 @@ dma: dma_lint
+		VERILOG_SOURCES="$(TIMER_SOURCES)"
+
+irq_lint:
+	verilator --lint-only -Wall -Wno-IMPORTSTAR \
+		--top-module interrupt_controller $(IRQ_SOURCES)
+
+irq: irq_lint
+	$(MAKE) MODULE=test_irq_ctrl TOPLEVEL=interrupt_controller \
+		VERILOG_SOURCES="$(IRQ_SOURCES)"
+
+# Run every SoC unit test (crossbar + register bank + AXI-Lite interconnect + SRAM + DMA + timer + IRQ).
 soc_all:
```

checkmake (0.3.2)

[warning] Target body for "soc_all" exceeds allowed length of 5 lines (7).

(maxbodylength)